### ISSUE TITLE
BCContainerHelper settings skipped

### ISSUE THREAD

**Issue by ronnykwon** [microsoft/AL-Go#1091]
I tried to add the following configurations in .github/AL-Go-Settings.json :
```
  "artifactDownloadTimeout" : 3600,
  "defaultDownloadTimeout" : 1800,
```

When I run the CI/CD Action I get this message : 
```
Import from C:\ProgramData\BcContainerHelper\6.0.17\BcContainerHelper\BcContainerHelper.ps1
BcContainerHelper version 6.0.17
Ignoring config setting artifactDownloadTimeout as the type in the config file is different than in the default configuration
Ignoring config setting defaultDownloadTimeout as the type in the config file is different than in the default configuration
BC.HelperFunctions emits usage statistics telemetry to Microsoft
Running on Windows, PowerShell 7.4.2
use settings and secrets
Analyzing repository
  Checking type
  Checking appFolders, testFolders and bcptTestFolders
  Application Dependency 24.0.0.0
  Checking artifact setting for project
  Downloading artifacts from https://bcartifacts-exdbf9fwegejdqak.b02.azurefd.net/sandbox/24.0.16410.20025/fr
  Downloading platform artifact /sandbox/24.0.16410.20025/platform
  Downloading C:\Users\Administrateur\AppData\Local\Temp\66f7a408-59c8-45d4-8f1e-652ee873[68](https://github.com/MyOrgPF/MyOrg_AL_Framework/actions/runs/9260111950/job/25473310589#step:8:70)ae.zip
  Could not download from CDN..., retrying from blob storage in 2 seconds...
```

And I will face the timeout issue.
```[tasklist]
### Tasks
```

**Comment by freddydk** [comment]
Duplicate of https://github.com/microsoft/navcontainerhelper/issues/3555

**Comment by freddydk** [comment]
Fixed with the latest containerhelper 6.0.18
